app/services/CadxResultService.scala

```diff
@@ -145,7 +154,20 @@ class CadxResultService @Inject()(
       given HeaderCarrier = HeaderCarrier()
       auditService.audit(auditEvent)
 
-      submissionRepository.save(submission.copy(state = Approved(fileName = state.fileName, reportingPeriod = state.reportingPeriod), updated = clock.instant())).map { _ =>
+      lazy val updatedInstance = clock.instant()
+      lazy val checksCompletedDateTime = EmailDateTimeFormatter.format(updatedInstance).replace("AM", "am").replace("PM", "pm")
+
+      val compResult = for {
+        subscription      <- subscriptionConnector.get(submission.dprsId)
+        platformOperator  <- OptionT(platformOperatorConnector.get(submission.dprsId, submission.operatorId)).getOrElseF(Future.failed(NoPlatformOperatorException(submission.dprsId, submission.operatorId)))
+      } yield emailService.sendSuccessfulBusinessRulesChecksEmails(Approved(fileName = state.fileName, reportingPeriod = state.reportingPeriod), checksCompletedDateTime, platformOperator, subscription)
+
+      compResult.onComplete {
```

Comment:
I don't think this is doing what you expect - compResult is actually a Future[Future[Done]], so if sending the email fails the outer future will still be successful (because of the way the email connector is *currently* written).  So in that scenario we'd log here that it had worked when it hadn't.

Reply:
For composition updated and now yields Done.